Sources/App/routes.swift

```diff
@@ -70,33 +70,18 @@ func routes(_ app: Application) throws {
             app.get(":owner", ":repository", ":reference", "**") {
                 try await packageController.documentation(req: $0, fragment: .root)
             }
-            app.get(":owner", ":repository", ":reference", "css") {
-                try await packageController.documentation(req: $0, fragment: .css)
-            }
```

Comment:
I believe these routes that have a matching `**` cousin aren't really used. IIUC, these are "browsing" routes that would resolve to `css/index.html` - which doesn't actually exist. So they should be safe to drop.

This could use some more testing/review but I'd like to trim the set of required routes down to a minimum before starting the transition to Pointfree's routing.

Reply:
Let's maybe make #1917 a part of this PR, too. I believe what's needed there is to focus the wildcard routes that we added for the `favicon` and I think there was another one, too, down to specific routes.